OsmAnd/src/net/osmand/plus/TargetPointsHelper.java

```diff
@@ -309,9 +309,7 @@ public TargetPoint getMyLocationToStart() {
 	public List<TargetPoint> getIntermediatePointsNavigation() {
 		List<TargetPoint> intermediatePoints = new ArrayList<>();
 		if (settings.USE_INTERMEDIATE_POINTS_NAVIGATION.get()) {
-			for (TargetPoint t : this.intermediatePoints) {
-				intermediatePoints.add(t);
-			}
+            intermediatePoints.addAll(this.intermediatePoints);
```

Comment:
Did the indentation (here and in the other places) get messed up or it is my impression?

Reply:
Yeah, I was just fixing the indentation now.